playbooks/roles/hammer_devel/tasks/hammer_install.yml

```diff
@@ -7,6 +7,12 @@
     remote: upstream
   with_items: "{{ hammer_devel_repositories }}"
 
+- name: 'Add fork remotes to cloned repositories'
```

Comment:
This currently isn't idempotent and really like to be able to run a command twice and not have it fail the second time.